content/applications/websites/ecommerce/managing_products/catalog.rst

```diff
@@ -0,0 +1,148 @@
+============================
+Organize the product catalog
+============================
+
+The eCommerce catalog is the equivalent of your physical store shelves: it allows customers to see
+what you have to offer. Clear categories, available options, sorting, and navigation threads help
+you structure it efficiently.
+
+Categorize the product catalog
+==============================
+
+In Odoo, there is a **specific category model** for your eCommerce. Using eCommerce categories for
+your products allows you to add a navigation menu on your eCommerce page. Visitors can then use it
+to view all products under the category they select. To do so, go to :menuselection:`eCommerce -->
+Products --> (Your product) --> Sales` and select the :guilabel:`Categories` you want under
+:guilabel:`eCommerce Shop`.
+
+.. note::
+   A single product can appear under multiple eCommerce categories.
+
+When your product is configured, go to the :guilabel:`Shop` page of your website and click on
+:menuselection:`Edit --> Customize`. In the :guilabel:`Categories` option, you can either enable a
+menu on the :guilabel:`Left`, on the :guilabel:`Top`, or both. The :guilabel:`Collapsable Category
+Recursive` button renders the *left* category menu collapsable.
+
+.. image:: catalog/catalog-panel-categories.png
+   :align: center
+   :alt: Categories options for your eCommerce website
+
+Browsing
+--------
+
+The eCommerce category is the first tool to organize and split your products. However, if you need
+an extra level of categorization in your catalog, you can activate various **filters** such as
+attributes or sort-by search.
+
+Attributes
+~~~~~~~~~~
+
+Attributes refer to **characteristics** of a product such as **color** or **material**, whereas
+variants are the different combinations of attributes. Attributes and variants can be found under
+:menuselection:`eCommerce --> Products --> (Your product) --> Attributes & Variants`.
```

Comment:
:menuselection:`Website app --> eCommerce --> Products` select your product and go to the  :guilabel:`Attributes & Variants` tab.